### PR TITLE
Added extra clothes to clothesmate vending machine

### PR DESCRIPTION
**Pull request by Alex-Selvey**
Added the extra clothes that Oracle added to the clothesmate vending machine

:cl: MemeProof
add: victorian clothes
add: military jackets
imageadd: icons and mob sprites for all Victorian clothing
/:cl:

added just the specific icons under the 
sunset\icons\obj\clothing
sunset\icons\mob

2018_09_22_03_26_07_sunsetstation_asteroid_center_xxiii

### CODE REVIEW

**Review by alexkar598 — DISMISSED** [review]
modularize your shit

**Comment by Wilchenx** [comment]
do these fit retrofuturistic 80's theme we are going for?

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -78,6 +78,9 @@
 		            /obj/item/storage/belt/fannypack = 1,
 		            /obj/item/storage/belt/fannypack/blue = 1,
 		            /obj/item/storage/belt/fannypack/red = 1,
+					/obj/item/clothing/suit/jacket/miljacket = 1,
```
Comment (ghost) [inline code/modules/vending/clothesmate.dm]: You can simply override/mirror this list in the `sunset folder

**Review by ghost — DISMISSED** [review]

**Comment by Alex-Selvey** [comment]
> do these fit retrofuturistic 80's theme we are going for?

It's just an option for us to put them in if we want. We can look into other clothing options to add to the clothesmate

**Review by ShattForte — DISMISSED** [review]
honk

**Comment by ghost** [comment]
Fix conflicts. There should be an automatic hook to do so with DMIs, to install it, run `tools/hooks/install.bat`, then git pull the upstream.

**Comment by Alex-Selvey** [comment]
installed hooks, but dmi merge keeps failing for me.
OSError: cannot identify image file '.merge_file_a15204'

**Comment by ghost** [comment]
Weird

File: code/modules/vending/clothesmate.dm [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	icon_deny = "clothes-deny"
 	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this free swag!;Why leave style up to fate? Use the ClothesMate!"
 	vend_reply = "Thank you for using the ClothesMate!"
-	products = list(/obj/item/clothing/head/that = 2,
+	/*products = list(/obj/item/clothing/head/that = 2, //disabled to not interfere with other changes in sunset\code\modules\vending\clothesmate
```
[the remark this answers is not in the document]
Comment (ghost) [reply code/modules/vending/clothesmate.dm]: if you're completely changing it, no. @MemeProof

[the remark this answers is not in the document]
Comment (ghost) [reply code/modules/vending/clothesmate.dm]: if you're completely changing it, just simply respecifing in the override
```
products = list(
...
)
``` 
will work

[the remark this answers is not in the document]
Comment (Alex-Selvey) [reply code/modules/vending/clothesmate.dm]: I tried overriding it, but I'll give it a go again. I probably just did it wrong the first few times I tried.

**Comment by ghost** [comment]
oh nice you got you some DMI conflicts.

only way to fix those is manually copy pasting

have fun

**Comment by Alex-Selvey** [comment]
> oh nice you got you some DMI conflicts.
> 
> only way to fix those is manually copy pasting
> 
> have fun

Do I need to just copy them to what is on master?
I tried the hooks, but they keep erroring.

**Comment by ghost** [comment]
Copy your new icons into the master dmis, yes

**Review by ghost — DISMISSED** [review]
I'm just going to ignore the newline diffs. Do delete the tmp/iconCache.sav, though

**Review by ghost — APPROVED** [review]

**Pull request merged**